CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres
 to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## UNRELEASED
+
+### Breaking changes
+* Removed properties `privatekey1`, `privatekey2`, `privatekey3`, `publickey1`, `marketplacekey`
```

Comment:
Changelog descriptions might need an reconsidered.

- Maybe we can point out that the different private keys were for testing purpose only. And class where it was removed from. I think it was the Unzer class?
- property file got removed completly.
- ApplepayUrls logic has changed again.